app/domain/donation.rb

```diff
@@ -33,6 +33,10 @@ def in_layer(layer)
     self
   end
 
+  def confirmations
+    @donations.joins(:invoice).group(:'invoices.recipient_id').sum('payments.amount')
```

Comment:
Ich weiss jetzt nicht ob wir das schon irgendwo haben, aber wenn es Rechnungen / Zahlungen in verschiedenen Währungen haben wird das so nicht korrekt klappen... Ist aber ein weiterführendes Problem, müssen wir wohl nicht heute lösen.